### PR TITLE
Story/173692 how users will see new announcements

### PR DESCRIPTION
**Pull request by pablocodeglia**
US:
[AB#173692](https://dev.azure.com/NHSiDev/InnovatorService/_workitems/edit/173692)

Task:
[AB#174470](https://dev.azure.com/NHSiDev/InnovatorService/_workitems/edit/174470)
[AB#174471](https://dev.azure.com/NHSiDev/InnovatorService/_workitems/edit/174471)

Screenshots:
- Login announcement (all users):
image

- Innovator: dashboard
image
- Innovator: innovation overview
image
- QA/A: dashboard
image
- Assessor: dashboard

image

### CODE REVIEW

File: src/modules/feature-modules/accessor/pages/dashboard/dashboard.component.ts [hunk 1]
```diff
@@ -54,6 +65,53 @@ export class DashboardComponent extends CoreComponent implements OnInit {
...
+          emptyMessageTitle: 'No innovations awaiting status assignment'
+        });
+      }
+
+      this.announcements = announcements;
+
+      this.setPageStatus('READY');
+    });
+
     this.statisticsService.getUserStatisticsInfo(qp).subscribe({
```
Comment (diogo-costa-bjss) [inline src/modules/feature-modules/accessor/pages/dashboard/dashboard.component.ts:115]: Shouldn't this be deleted?

Comment (pablocodeglia) [reply src/modules/feature-modules/accessor/pages/dashboard/dashboard.component.ts]: Yes it should! Done.

File: src/modules/theme/components/announcements/announcement-card.component.html [hunk 2]
```diff
@@ -1,9 +1,24 @@
...
+        </div>
+      </div>
+      <p *ngIf="announcementCardData.innovations && !isOverviewPage" class="font-color-secondary">
+        <span
+          >{{ "Re:" }}
+          <ng-container *ngFor="let innovation of announcementCardData.innovations; let first = first; let last = last">
+            <span *ngIf="!first && !last">{{ ", " }}</span>
+            <span *ngIf="last">{{ " and " }}</span>
+            <span>{{ innovation }}</span>
+          </ng-container>
```
Comment (diogo-costa-bjss) [inline src/modules/theme/components/announcements/announcement-card.component.html:19]: I don't really like us doing this here. Would prefere to either:
- Do this on the TS, and create a variable with it.
- Create a pipe that will contain this logic and can be reused. I can see this be used in other places. Would contain the logic to display:
    - [A] -> A
    - [A,B] -> A and B
    - [A,B,C] -> A, B and C

**Review by diogo-costa-bjss — APPROVED** [review]

File: src/modules/theme/components/announcements/announcement-card.component.html (hunk 2 above)
Comment (pablocodeglia) [reply src/modules/theme/components/announcements/announcement-card.component.html]: Yes sounds like a good idea, will do the TS version, and later try to create a pipe, I think we do have other cases of similar cases.

**Pull request merged**